crucible-llvm/src/Lang/Crucible/LLVM/Translation/Types.hs

```diff
@@ -111,7 +111,7 @@ llvmTypeAsRepr xs f = go (llvmTypeToRepr xs)
        go []       = f UnitRepr
        go [Some x] = f x
 
-       go _ = error $ unwords ["llvmTypesAsRepr: expected a single value type", show xs]
+       go _ = error $ unwords ["llvmTypeAsRepr: expected a single value type", show xs]
```

Comment:
This looks like a place we could use `panic`

Reply:
Agreed, but I don't think it should hold up this PR.